src/main/java/org/opensearch/ad/cluster/ADClusterEventListener.java

```diff
@@ -143,7 +144,7 @@ public void clusterChanged(ClusterChangedEvent event) {
                                     LOG.info(REMOVE_MODEL_MSG + " {}", modelId);
                                     modelManager
                                         .stopModel(
-                                            modelManager.getDetectorIdForModelId(modelId),
+                                            SingleStreamModelIdMapper.getDetectorIdForModelId(modelId),
```

Comment:
Looks like this method was moved from `ModelManager` to `SingleStreamModelIdMapper`, but don't see it removed in this PR. Is it removed in a different PR?

Reply:
And can you confirm that the usages of the methods moved to `SingleStreamModelIdMapper` have been updated in a different PR?

Reply:
yes, it is updated in https://github.com/opensearch-project/anomaly-detection/pull/221